03/app.js

```diff
@@ -0,0 +1,11 @@
+const buttonList = document.querySelectorAll('button');
+
+buttonList.forEach(function (element) {
```

Comment:
Tutaj już nie trzeba bo mamy tablicę. Jeśli jest pusta to po prostu `.forEach` się nie wykona.